icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -49,8 +54,40 @@ def test_download_granules_with_subsetting(reg, session):
     reg.download_granules(path)
 
 
-# def test_download_granules_without_subsetting(reg_a, session):
-#     path = './downloads'
-#     reg_a.order_granules(session, subset=False)
-#     reg_a.download_granules(session, path)
-#     #check that the max extent of the downloaded granules isn't subsetted
+def test_download_granules_without_subsetting(reg, session, capsys):
+    """
+    Test that granules can be ordered from NSIDC and downloaded with the `subset=False`
+    option.
+    """
+    path = "./downloads"
+
+    reg.order_granules(verbose=False, subset=False, email=False)
+    out, err = capsys.readouterr()  # capture stdout and stderr
+    assert out.startswith(
+        "Total number of data order requests is  1  for  3  granules.\n"
+        "Data request  1  of  1  is submitting to NSIDC\n"
+    )
+    assert err == ""
+
+    assert reg.reqparams == {
```

Comment:
Anyone know why codecov gives "not covered by tests" warnings for the tests?

Reply:
Hmm, the tests should be covered if they ran. I can try again later when changing from travis to gh actions and see if it changes things.

Reply:
Ah, I think the codecov report wasn't uploaded because the travis test failed (one of the assert tests wasn't deterministic, but I fixed that in 82fa916d176619476668749902e15674b10c5fd1). I've switched to GitHub Actions and am re-running the integration test again.